### PR TITLE
Update: basic stats api

### PR DESCRIPTION
**Pull request by dpaez**
[seeder-core] Exposing some basic stats API 

Related to #13

### CODE REVIEW

File: packages/seeder/src/seeder.js [hunk 1]
```diff
@@ -81,6 +119,56 @@ class Seeder extends EventEmitter {
...
+    async function getCoreStats (core) {
+      if (!core) return {}
+      const stats = core.stats
+      const networkingStats = {
+        key: core.key,
+        discoveryKey: core.discoveryKey,
+        peers: core.peers.length
+      }
+      return {
+        ...networkingStats,
```
Comment (estebanprimost) [inline packages/seeder/src/seeder.js]:
```suggestion
      return {
        key: core.key,
        discoveryKey: core.discoveryKey,
        peers: core.peers.length,
```

File: packages/seeder/src/seeder-demo.js [hunk 2]
```diff
@@ -2,11 +2,16 @@ const Seeder = require('./seeder')
...
 const main = async () => {
   const s = new Seeder()
-  await s.seed(['aa27834b41fa78c091ddb066e889202775bb3b5f57586c69d709be9f7231fccf'])
+  await s.seed(['2a09008a55c8f4bcc14987ed43331675611a2b506859ebfdd050b0f41746ad02'])
   s.on('drive-progress', (...args) => console.log('progress', args))
-  s.on('drive-finish', (dkey, ...args) => {
+  s.on('drive-finish', async (dkey, ...args) => {
     console.log('finish download', dkey)
     console.log(`Content available in: ${s.opts.storageLocation}`)
+    const stats = await s.allStats()
```
Comment (estebanprimost) [inline packages/seeder/src/seeder-demo.js]: Why not call to `stat(dkey)` here?

**Review by estebanprimost — CHANGES_REQUESTED** [review]

File: packages/seeder/src/seeder-demo.js (hunk 2 above)
Comment (dpaez) [reply packages/seeder/src/seeder-demo.js]: Yeah, that would be better. I was just toying around with `allStats()` method.

**Review by estebanprimost — APPROVED** [review]

File: packages/seeder/src/seeder.js [hunk 3]
```diff
@@ -75,29 +116,123 @@ class Seeder extends EventEmitter {
...
-      return this.downloads
+      await this.networker.configure(discoveryKey, { announce: this.opts.announce, lookup: this.opts.lookup })
+      const unmirror = drive.mirror()
+      this.mirrors.set(keyString, unmirror)
+      const unwatch = drive.watch('/', () => {
+        this.onEvent('update')
+      })
+
+      const contentFeed = await new Promise((resolve, reject) => {
+        drive.getContent((err, cf) => {
```
Comment (estebanprimost) [inline packages/seeder/src/seeder.js]: Maybe [`pify`](https://github.com/sindresorhus/pify) helps you here.

**Review by estebanprimost — APPROVED** [review]

**Pull request merged**